spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -396,7 +402,18 @@ def main(args=None):
         # apply laplacian filtering
         if laplacian:
             printv('\nApply Laplacian filter...', verbose)
-            run_proc(['sct_maths', '-i', 'data_straightr.nii', '-laplacian', '1', '-o', 'data_straightr.nii'], verbose)
+            img = Image("data_straightr.nii")
+
+            # apply std dev to each axis of the image
+            sigmas = [1 for i in range(len(img.data.shape))]
+
+            # adjust sigma based on voxel size
+            sigmas = [sigmas[i] / img.dim[i + 4] for i in range(3)]
```

Comment:
See my earlier comment about this adjustment. ("Should this be a part of the `laplacian` function, since we do it every time?")